build/_k.shade

```diff
@@ -1,7 +1,7 @@
 @{/*
```

Comment:
No, it will break a lot of things. We want to make sure the changes that are more visible to external users are checked in for beta4 milestone ASAP. For those stuff mainly for internal use, we leave it for now, so that they don't block more important changes.